website/content/docs/agent/telemetry.mdx

```diff
@@ -551,6 +552,23 @@ Historically, `internal` RPC operation metrics were accounted under the same met
 
 The `unreported` value for the `request_type` in the table above refers to RPC requests within Consul where it is difficult to ascertain whether a request is `read` or `write` type.
 
+The `unreported` value for the `leader` label in the table above refers to RPC requests where Consul cannot determine the leadership status for a server.
+
+#### Read Request Labels
+
+In addition to the labels above, for read requests, the following may be populated:
+
+| Label Name                            | Description                                                                    | Possible values                                            |
+| ------------------------------------- | ------------------------------------------------------------------------------ | ---------------------------------------------------------- |
+| `blocking`                            | Whether the read request passed in a `MinQueryIndex`.                          | `true` if a MinQueryIndex was passed, `false` otherwise.   |
+| `target_datacenter`                   | The target datacenter for the read request.                                    | The string value of the target datacenter for the request. |
+| `locality`                            | Gives an indication of whether the RPC request is local or has been forwarded. | `local` or `forwarded`.                                    |
+
+##### Read Request Label Explanations
+
+We determine the values for the `locality` label by looking at the current server datacenter and the request target datacenter. If they match, we call that `local` RPC request.
+Otherwise, we call that RPC request `forwarded`.
```

Comment:
Let's rephrase this to avoid first person language.  Suggestion:

"The values for the `locality` label are determined by comparing the current server datacenter and the request target datacenter.  When those values are equal, the RPC request locality is `local`, otherwise locality is `forwarded`.

Alternately, remove this section and just add to the table above in the Possible values column:

`local` if current server data center is the same as target_datacenter, otherwise `forwarded`.